forch/file_handlers.py

```diff
@@ -0,0 +1,25 @@
+"""File changes watcher"""
```

Comment:
It's not clear what abstraction you're going for here... Is this file meant to be specific to Faucet config files (has faucet-specific file stuff in it), or generic for watching changes in any file?

Reply:
It is specific to faucet config files but we may also put other file handlers in this file.

Reply:
I think you should commit either way -- straddling the fence like this just leads to spagetti since you don't have clean abstraction either way. If a more generic something is needed later, it's better to do the refactor as needed. So -- should either be a generic thing while the faucet-specific stuff is in forchestrator.py, or a faucet-specific thing...